src/test/java/com/google/javascript/clutz/partial/date_like.js

```diff
@@ -0,0 +1,12 @@
+goog.module('date.like')
+goog.forwardDeclare('goog.date.Date');
```

Comment:
Add a comment with `//!!` that forward declare is there to match real issue in closure/date_like, but not fully relevant to the test. I.e. it should work the same with a regular goog.require.

Reply:
//!! comments are ignored during golden checking.